bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/config/ModelSyncGradleSettings.kt

```diff
@@ -76,6 +78,14 @@ data class SyncDirection(
         (includedModulePrefixes as MutableSet).add(prefix)
     }
 
+    fun excludeModule(module: String) {
```

Comment:
This API is used in Gradle scripts. API documentation would be useful here. The user then does not have to go to the docs to read what this does and how it interacts with included modules.

Reply:
I can see the benefit of having the KDoc comment but if we do that we should do it consistently for everything (bulk-sync and api-gen) and find a way to ensure it is consistent with the docs. That's out of scope for this PR in my eyes.

Reply:
Ok, I understand that.

Reply:
We could just start with the methods that you created or touched, i.e. [Boy scout rule](https://innolution.com/resources/glossary/boy-scout-rule).